### DISCUSSION TITLE
Welcome to authme Discussions!

### DISCUSSION THREAD

**Discussion by Levminer** [Levminer/authme#38] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EricBDuncan** [comment]
Will authme work as a portal app on a USB or Network drive for multi-pc use?

**Reply by Levminer** [reply]
Yes, but the settings and codes are stored in `%appdata%`. So you have to import your codes on every computer.

**Comment by katastrophe92** [comment]
Hello authme team & community! Thanks for the amazing project! Is planned also support for browser extension or android? Kind regards!

**Reply by Levminer** [reply]
The mobile support is tracked at #286 

A browser extension is currently not planned.

**Comment by Sisco95** [comment]
How are we notified when there are new releases or necessary updates that need to be installed?

**Reply by Levminer** [reply]
If you are on Windows you will see a banner to download the new update. On Linux/Mac you can download the update manually.

**Reply by Sisco95** [reply]
We still get this banner even if the app is offline?

**Reply by Levminer** [reply]
> We still get this banner even if the app is offline?

No, an internet connection is required.

**Reply by Sisco95** [reply]
Perfect. How often do you update the app or is there no set schedule? How do I know if the app is connected to the internet or not?

**Reply by Levminer** [reply]
There is an update on the 2nd Tuesday of every month (Patch Tuesday). The app checks for an update when you start it.

**Reply by Sisco95** [reply]
Perfect on the patch schedule. So as long as the app is closed it will be disconnected from the internet. Got it.

**Comment by OFandoud** [comment]
Hello,

I have an issue when importing from 2fa to authme get the below error.

[AUTHME ERROR] (2024. 06. 21. 15:28:03) Unknown runtime error occurred: TypeError: Cannot read properties of undefined (reading 'replaceAll')

**Reply by Levminer** [reply]
Hello,

Sorry for the late respone. Can you send you file you are trying to import? Just replace the secret values with xxxx or something.

**Comment by Sisco95** [comment]
If I put a password on the authme app and forgot, is there a way to remove that password or do I need to reinstall the program?

**Reply by Levminer** [reply]
You need to delete Authme's config folder. On Windows: %LocalAppData%/com.levminer.authme

**Reply by Sisco95** [reply]
Thank you sir.

**Reply by rhiller-png** [reply]
Sorry to resurrect .Anyone know what files this would be on a Mac? Have deleted/reinstalled the app as well as deleted the ~/Library/Prefereces/com.leviner file... as well as ~/Library/Caches/com.leviner.authme file. After reinstalling, it's still asking for a passoword. Ideas?
